### PR TITLE
drop support for EOL rubies

### PR DESCRIPTION
**Pull request by grosser**

### CODE REVIEW

**Comment by Skarlso** [comment]
Well, this failed horribly. :)

**Comment by grosser** [comment]
build still pending ... not sure why

**Comment by Skarlso** [comment]
```
[!] There was an error parsing `Gemfile`: 
[!] There was an error while loading `jsonpath.gemspec`: syntax error, unexpected `end', expecting end-of-input. Bundler cannot continue.
```

File: jsonpath.gemspec [hunk 1]
```diff
@@ -5,10 +5,10 @@ require File.join(File.dirname(__FILE__), 'lib', 'jsonpath', 'version')
 Gem::Specification.new do |s|
   s.name = 'jsonpath'
   s.version = JsonPath::VERSION
-  if s.respond_to? :required_rubygems_version=
-    s.required_rubygems_version =
+  s.required_rubygems_version =
       Gem::Requirement.new('>= 0')
   end
```
Comment (Skarlso) [inline jsonpath.gemspec:10]: @grosser Remove this `end`

**Comment by grosser** [comment]
all good now except the ruby-head failures

**Comment by Skarlso** [comment]
I fixed that. I'll be home soon and merge it then you can rebase and we'll
merge this pr.

**Comment by Skarlso** [comment]
@grosser Please merge master and re-run this.

**Comment by grosser** [comment]
jruby still unhappy ... sane as master

**Comment by Skarlso** [comment]
Oh yeah, I know. Ignore jruby for now. It has some kind of problem with the Rakefile. ¯\_(ツ)_/¯

**Pull request merged**